src/styles/main.scss

```diff
@@ -377,6 +377,12 @@ header.navbar .navbar-brand {
   }
 }
 
+// Custom styles for ng-select validation
+ng-select.ng-invalid.ng-touched .ng-select-container {
+  border-color: #dc3545;
```

Comment:
Can we here using some variable from attached theme or bootstrap variables?

Upd.
```
$red:     #dc3545 !default;
```

Reply:
Actual `$red` color is different from validation colors, so it was changed to project one.